frontend/src/store/store.js

```diff
@@ -117,6 +132,9 @@ try {
       tokenTime: null,
       roles: [],
       newsletterState: null,
+      gmsAllowed: null,
+      gmsPublishName: null,
+      gmsPublishLocation: null,
```

Comment:
Does it make sense to define here the default values for the gmsPublish-Attributes?

Because as i see the default settings in the graphql-model are removed...

Reply:
Wenn der User sich einloggt, wird die Funktion login weiter oben aufgerufen und kopiert die Werte vom User Entity in den Store. Wenn der User neu angelegt wurde, sind das die default Werte vom entity ansonsten sind es die Werte, welcher der User in der Vergangenheit eingestellt hat. 
Also nein, default Werte machen hier keinen Sinn. 

Bei den anderen Werte wurden auch keine default Werte gesetzt.

Reply:
Aber sind die GMS-Attribute hier nicht eine Ausnahme was die Default-Werte angeht?
Die anderen Attribute müssen ja auch nicht mit Default-Werten vorbelegt sein.
Aber ich glaub dir jetzt mal, dass es auch ohne Defaults funktioniert...